src/plumtree_broadcast.erl

```diff
@@ -117,6 +125,10 @@
 %% to generate membership updates as the ring changes.
 -spec start_link() -> {ok, pid()} | ignore | {error, term()}.
 start_link() ->
+    LazyTickPeriod = application:get_env(plumtree, lazy_tick_period,
```

Comment:
you're probably thinking of `partisan_config`, in `plumtree` simple `application:get_env` is used instead